virttest/qemu_devices/qdevices.py

```diff
@@ -3151,6 +3151,21 @@ def initialize(self, cpuinfo):
                     if next_addr[0] % self.vcpus_count != 0:
                         self.reserve(next_addr)
                     next_addr = self._increment_addr([None], next_addr)
+        elif cpuinfo.qemu_type == "arm64" in cpuinfo.qemu_type:
```

Comment:
```suggestion
        elif cpuinfo.qemu_type == "arm64":
```
Shall we need the second operator `in` here?

Reply:
Actually, this logic arm is not currently used. On arm we use `qdevices.QStrictCustomBus` instead of `qdevices.QCPUBus`.
So after thinking about it, I removed this part code, and let's consider it again when there are specific usage scenarios in the future.